Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -413,11 +353,11 @@ - (void)profileWhitelistDidChange:(NSNotification *)notification
     NSString *_Nullable recipientId = notification.userInfo[kNSNotificationKey_ProfileRecipientId];
     NSData *_Nullable groupId = notification.userInfo[kNSNotificationKey_ProfileGroupId];
     if (recipientId.length > 0 && [self.thread.recipientIdentifiers containsObject:recipientId]) {
-        [self ensureDynamicInteractions];
+        [self.conversationViewModel ensureDynamicInteractions];
```

Comment:
Eventually, I hope we can do away with "dynamic interactions" entirely and make them:

* "ephemeral/transient" (never persisted in DB)
* created/updated by the CVM.